packages/neon-api/src/provider/common.ts

```diff
@@ -61,3 +61,21 @@ export function findGoodNodesFromHeight(
   const threshold = bestHeight - tolerance;
   return sortedNodes.filter((n) => n.height >= threshold);
 }
+
+export interface Entry {

```

Comment:
What `Entry` does this describe? Please have a name that is more specific that just entry.

Reply:
in api response exist the entries field, which is an array, so entry represents one information these array

Reply:
I meant something more specific. Like naming it `AddressAbstractTransactionEntry` that indicates its purpose so I do not have to try to figure out where it belongs.